### PR TITLE
Nataj branch

### PR DESCRIPTION
**Pull request by sarahnataj**
Second time submission, I removed the picture, since I was thinking this is not working because of this.

### CODE REVIEW

**Comment by sarahnataj** [comment]
Why d

**Comment by ccontrer** [comment]
I think after line `48`, you should have `---` in a separate line. Then your bio in a new line. Also, you can get rid of the `>` in line `53`.

File: content/authors/natajs/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  link: "sarah.nataj@gmail.com"
+- icon: linkedIn
+  link: "https://www.linkedin.com/in/sarahnataj/"
+- icon: github
+  link: "https://github.com/sarahnataj"
+
+
+user_groups:
+- committee: members
+  role: Project member: Fotech Solutions
```
Comment (ccontrer) [inline content/authors/natajs/index.md:47]: add `---` after this line

File: content/authors/natajs/index.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+Display name
```
Comment (ccontrer) [inline content/authors/natajs/index.md:1]: add `---` before this line

**Comment by ianabc** [comment]
There were a few little niggly things wrong in the `_index.md`. They would have been awkward to sort out so I've made a few more commits to fix them up, and I think this is ready to merge now. Basically YAML is _really_ fussy about syntax and whitespace, if you're interested in comparing the changes you can look at [the diff](https://github.com/sarahnataj/m2pi.ca/compare/b0f091d9300ce1993d9eaa3817a6f87818c6fe6d...sarahnataj:Nataj_Branch).

**Comment by ianabc** [comment]
Looks good to me, let's merge!

**Pull request merged**